2_process/src/process_fcn.R

```diff
@@ -1,17 +1,18 @@
 #' # Define process() function
 #' 
-#' The `process()` function filters the data and assigns colors/pch values based on
+#' @description The `process()` function filters the data and assigns colors/pch values based on
 #' model_type.
 #' 
-#' @param input_data chr, input object (raw data)
+#' @param input_file data.frame, input object (raw data)
```

Comment:
Ah, ok, so I think we are flipping these. In my mind, I would choose from either `@param input_file chr, the character string representing the file containing the raw data` OR `@param input_data data.frame, input object (raw data)`. Based on the rest of your function, you are passing in the character string to the file, not the data.frame itself, because you later call `read_csv` to load in the data from the file. So, I think you want to call this `input_file` (as you changed it to) but it is a `chr`

Reply:
Yes! I meant to double check that I had that described correctly after changing the other function descriptions. Thanks! I cleared this up.